bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapResourceLink.java

```diff
@@ -0,0 +1,135 @@
+package de.agilecoders.wicket.core.markup.html.bootstrap.button;
+
+import org.apache.wicket.IResourceListener;
+import org.apache.wicket.markup.html.link.Link;
+import org.apache.wicket.markup.html.link.ResourceLink;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.request.cycle.RequestCycle;
+import org.apache.wicket.request.handler.resource.ResourceReferenceRequestHandler;
+import org.apache.wicket.request.mapper.parameter.PageParameters;
+import org.apache.wicket.request.resource.IResource;
+import org.apache.wicket.request.resource.ResourceReference;
+import org.apache.wicket.request.resource.IResource.Attributes;
+
+import de.agilecoders.wicket.core.markup.html.bootstrap.button.Buttons.Type;
+
+/**
+ * 
+ * Same as default {@link ResourceLink} but based on {@link BootstrapLink} which is styled by bootstrap. 
+ * 
+ * @author ymlaine
+ *
+ */
+public class BootstrapResourceLink extends BootstrapLink<Void> implements IResourceListener {
```

Comment:
I think there is no need of such component in the library.
You can just use: `new ResourceLink(...).add(new ButtonBehavior(...))`.
Copying big chunks of code from Wicket to the library begs for problems in the maintainance in the future